examples/kitchensink/app-counter/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Kitchensink counter app in Frint",
   "main": "index.js",
   "scripts": {
-    "build": "webpack",
+    "build": "../../../node_modules/.bin/webpack",
```

Comment:
if doing so then let's remove the reference to Webpack in this 'package.json' file